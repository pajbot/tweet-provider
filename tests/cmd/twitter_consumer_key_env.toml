bin.name = "tweet-provider"

status.code = 0

[env.add]
TWEET_PROVIDER_DUMP_ARGS_AND_EXIT = "1"
PAJBOT_TWITTER_CONSUMER_KEY = "test"
