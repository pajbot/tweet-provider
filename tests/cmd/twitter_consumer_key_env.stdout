Args {
    config_path: "tweet-provider.toml",
    config: Config {
        websocket: WebSocket {
            listen_addr: 127.0.0.1:2356,
        },
        twitter: Twitter {
            consumer_key: Some(
                "test",
            ),
            consumer_secret: None,
            access_token: None,
            access_token_secret: None,
            always_restart: false,
        },
    },
    log_level: Info,
    log_timestamps: UTC,
}
